src/utils.js

```diff
@@ -131,7 +131,7 @@ export function verifyIfObjectsMatch(matcher = {}, verify = {}, exact = false) {
     }
 
     keys(matcher).forEach((key) => {
-        if (verify.hasOwnProperty(key)) {
+        if (verify !== null && verify.hasOwnProperty(key)) {
```

Comment:
Since the loop does nothing if verify is undefined, could you move this null check out of the forEach and return false? And if you could add a test for this please.